src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java

```diff
@@ -0,0 +1,178 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.amazonaws.regions.Region;
+import com.amazonaws.services.datapipeline.model.Field;
+import com.amazonaws.services.datapipeline.model.PipelineObject;
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import com.google.common.collect.Lists;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Creates a Data Pipeline for backing up Dynamo DB tables within a region.
+ */
+public class DynamoDataPipelineHelper {
+
+    private static final String READ_THROUGHPUT_PERCENTAGE = "0.3";
```

Comment:
This controls how much of the current RCU the EMR job is allowed to consume. It's not clear to me how we should set this ratio right now, can we try it with 0.3 for now to see how it goes in UAT?

I think first we want to properly scale the tables to handle peak production traffic. Second, we should pick a time window for backup where there's low traffic and then set this throughput ratio high enough that the backup finishes quickly (and before prod traffic picks up again), but low enough it won't brown out production traffic.

This also relates to the issue you brought up of ever increasing append-only tables, since that will quickly increase the backup time. I'll create a separate task for Upload, Attachment, Record tables (there's a couple approaches to consider)